src/components/inline-grid/index.js

```diff
@@ -0,0 +1,37 @@
+// @vendors
+import { array } from "prop-types";
+
+// @styles
+import styles from "./styles.module.scss";
+
+const InlineGrid = ({ data }) => {
+  return (
+    <section className={styles.container}>
+      {data?.map((item) => {
+        const { name, main_image: mainImg } = item.data;
+        const imgStyle = {
```

Comment:
nit: It would be better for optimization and seo to use <img> instead of backgroundImage if possible